scene/2d/camera_2d.h

```diff
@@ -47,10 +47,11 @@ class Camera2D : public Node2D {
 		CAMERA2D_PROCESS_IDLE
 	};
 
-protected:
-	Point2 camera_pos;
-	Point2 smoothed_camera_pos;
-	bool first = true;
+private:
+	bool initialised;
```

Comment:
I think we tend to prefer US spelling, hence `initialized`. I actually prefer the US variant, even being from UK myself. :slightly_smiling_face:

Reply:
Changed to `initialized`.